### PR TITLE
SOLID-388: Add SMS Button and Update Social Button SVG Icons

### PR DESCRIPTION
**Pull request by allisonchefec**
- added SMS button to social buttons. 
- updated svgs for all social buttons to reflect new icon set.

### CODE REVIEW

File: docs/buttons.html [hunk 1]
```diff
@@ -275,53 +286,69 @@ <h2 class="bold xs-mb2">Social Buttons</h2>
...
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 38">
+        <title>Instagram</title>
+        <path d="M31.1 0H7C2.8 0 0 2.8 0 7v24c0 4.2 2.8 7 7 7h24.1c4.2 0 6.9-2.8 6.9-7V7c0-4.2-2.8-7-6.9-7zM33 3.9h.6v6.7h-6.4V3.9H33zM13.4 15c1.3-1.7 3.3-2.9 5.6-2.9s4.3 1.1 5.6 2.9c.8 1.1 1.3 2.5 1.3 4 0 3.8-3.1 6.9-6.9 6.9s-6.9-3.1-6.9-6.9c0-1.5.5-2.9 1.3-4zm21.7 16c0 2.1-1.9 4-4 4H7c-2.1 0-4-2-4-4V14.6h6.1c-.5 1.5-.8 2.8-.8 4.2 0 5.9 4.8 10.8 10.7 10.8 5.9 0 10.7-5 10.7-10.9 0-1.4-.3-2.7-.8-4.2H35V31z"/>
       </svg>
       Follow
     </button>
+
+    <button class="button button--sms button--icon xs-mr1 xs-mb1" href="">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 38">
+        <title>Speech filled</title>
```
Comment (lindseymaratta) [inline docs/buttons.html]: I'd change the title here to "SMS". However, in normal implementation, this title should really be "Share via SMS" and the `svg` element should include the `aria-hidden="true" ` attribute to hide the decorative icon from screen reader. I wonder if we should just do that here with all of them as an example of proper use?

**Comment by lindseymaratta** [comment]
I added a comment to line 304 in buttons.html but i'll repeat it here: 

I'd change the title here to "SMS". However, in normal implementation, this title should really be "Share via SMS". In this case, the svg element should include the aria-hidden="true" attribute to hide the decorative icon from screen reader. I wonder if we should just do that here with all of them as an example of proper use?

**Comment by allisonchefec** [comment]
thanks @lindseymaratta accessibility queen. made your suggested edits!

**Comment by allisonchefec** [comment]
ah good catch @emilybrick 

updated.

**Comment by allisonchefec** [comment]
I've updated this, can I get an approval pleeeease?

**Pull request merged**